packages/core/admin/admin/src/pages/HomePage/CloudBox.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { useTracking } from '@strapi/helper-plugin';
+import { Box, Flex, Stack, Typography } from '@strapi/design-system';
+import cloudIconBackground from './assets/strapi-cloud-background.png';
+import cloudIcon from './assets/strapi-cloud-icon.svg';
+import cloudFlags from './assets/strapi-cloud-flags.svg';
+
+const BlockLink = styled.a`
+  text-decoration: none;
+`;
+
+const CloudCustomWrapper = styled.div`
+  width: 56px;
+  height: 56px;
+  background-image: url(${({ src }) => src});
+  padding: 11px;
+  margin-right: ${({ theme }) => theme.spaces[6]};
+`;
```

Comment:
Instead of using `styled.div` I'd suggest to use `styled(Box)` -  this will allow you to get rid of most of this CSS and present it through props.

Instead of using margin-right I think you can use `gap` on the parent element.

The padding should probable use `theme.spaces[3]` instead, because our grid is based on the magic number 4.